File: tags/Build_5.1.136.Weekly/Drivers and Simulators/FilterWheel Simulator .NET/FilterWheelServer/frmMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace ASCOM.FilterWheelSim
{
    public partial class frmMain : Form
    {
        delegate void SetTextCallback(string text);

        public frmMain()
        {
            InitializeComponent();
        }

    }
}

File: tags/Platform6_Alpha18/DriverTemplates/LocalServer Composite Driver VB/frmMain.vb

Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace TEMPLATENAMESPACE
	Public Partial Class frmMain
		Inherits Form
		Delegate Sub SetTextCallback(text As String)

		Public Sub New()
			InitializeComponent()
		End Sub

	End Class
End Namespace

'=======================================================
'Service provided by Telerik (www.telerik.com)
'Conversion powered by NRefactory.
'Built and maintained by Todd Anglin and Telerik
'=======================================================
